Azure/sonic-buildimage
# Add docker-router-advertiser to support IPv6 router advertisements

**- What I did**

  - Add new docker, `docker-router-advertiser` into which we install radvd and all necessary configuration files to listen for IPv6 router solicitations and send IPv6 router advertisements on all downlink VLAN interfaces which are configured with IPv6 addresses.

**- How to verify it**

  1. Build and install new image on a device
  2. Ensure docker-router-advertiser starts at boot via systemd
  3. Listen on VLAN interfaces for router advertisements sent at periodic (< 10 minute) intervals

**- A picture of a cute animal (not mandatory but encouraged)**

```
                          /\
                  __     /^|
                 / /\###//||
                 |//  #  \//
                  :(o) (o);
                   \     /##
                    || ||  ##
                    /   \   `--...____
                   ( @_@ )
                    `---'    /
                     |      |
                      \      \  
```
